v2/spanner-common/src/main/java/com/google/cloud/teleport/v2/spanner/migrations/schema/cassandra/SourceTable.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (C) 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.google.cloud.teleport.v2.spanner.migrations.schema.cassandra;
+
+import com.google.auto.value.AutoValue;
+import com.google.common.collect.ImmutableList;
+import java.io.Serializable;
+import java.util.List;
+
+/** Represents a source table schema with its name and associated columns. */
+@AutoValue
+public abstract class SourceTable implements Serializable {
```

Comment:
do we need this parallel structure ?

Reply:
@bharadwaj-aditya we are using this same as DDL Object where we are populating Table and Column based on Resultset

Reply:
why can we not use the existing structure of srcSchema to populate based on resultset?

Reply:
we can also do the same, we have created POJO for code readability and also for future extensibility 
Let me remove if it is creating confusion @shreyakhajanchi

Reply:
@shreyakhajanchi removed the parallel structure which was leading confusion